sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/BenchmarkRequestSubscriber.java

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.benchmark;
+
+import com.codahale.metrics.Meter;
+import com.codahale.metrics.Timer;
+import org.reactivestreams.Subscription;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import reactor.core.publisher.BaseSubscriber;
+
+import java.util.concurrent.Semaphore;
+import java.util.concurrent.atomic.AtomicLong;
+
+public class BenchmarkRequestSubscriber<T> extends BaseSubscriber<T> {
+    final static Logger logger = LoggerFactory.getLogger(BenchmarkRequestSubscriber.class);
+    private Meter successMeter;
+    private Meter failureMeter;
+    private Semaphore concurrencyControlSemaphore;
+    private AtomicLong count;
+    public Timer.Context context;
```

Comment:
public?

Reply:
now AsyncCtlWorkload moved to different package we need to access this, we could have use setter , but this is more simple  given its usecase.